OrbitClientGgp/ClientGgp.cpp

```diff
@@ -164,6 +180,46 @@ bool ClientGgp::InitCapture() {
   return true;
 }
 
+void ClientGgp::InformUsedSelectedCaptureFunctions(
+    absl::flat_hash_set<std::string> capture_functions_used) {
+  if (capture_functions_used.size() != options_.capture_functions.size()) {
+    for (const std::string selected_function : options_.capture_functions) {
+      if (!capture_functions_used.contains(selected_function)) {
+        ERROR("Function matching %s not found; will not be hooked in the capture",
+              selected_function);
+      }
+    }
+  } else {
+    LOG("All functions provided had at least a match");
+  }
+}
+
+std::string ClientGgp::SelectedFunctionMatch(const FunctionInfo& func) {
+  for (const std::string selected_function : options_.capture_functions) {
```

Comment:
Sorry for nit picking:
`for (const std::string& selected_function : options_.capture_functions) {`
to avoid the copy

Reply:
Oh right, thanks! I have updated it here and in other places in the code as well